docs/_packages/socket_mode.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Socket Mode
+permalink: /socket-mode
+order: 7
+anchor_links_header: Usage
+---
+
+# Slack Socket Mode
+
+
+## Installation
+
+```shell
+$ npm install @slack/socket-mode
+```
+
+---
+
+### Initialize the client
+
+The package is designed for the newly released [**Socket Mode**](https://api.slack.com/socket-mode) feature. It exports a `SocketModeClient` class. Your app will create an instance of the class for each workspace it
+communicates with. Creating an instance requires an **app level token** from Slack. Apps connect to the **Socket Mode** API using the newly introduced **app level token**, which starts with `xapp`.
```

Comment:
Is there a line break between this and the last??? If so, could it be removed?

Reply:
In the other guide, you use "App Token". Regardless of what is used, could we make it consistent across the guides?

The API site uses "app-level token"